src/msg/async/AsyncMessenger.cc

```diff
@@ -580,7 +339,16 @@ int AsyncMessenger::bind(const entity_addr_t &bind_addr)
 
   // bind to a socket
   set<int> avoid_ports;
-  int r = processor.bind(bind_addr, avoid_ports);
+  int r = 0;
+  unsigned i = 0;
+  for (auto &&p : processors) {
+    r = p->bind(bind_addr, avoid_ports);
+    if (r < 0) {
+      assert(i == 0);
```

Comment:
this is related to local listen table. Posix backend shares listen table in the kernel, so all threads can use the same listen table naturally. Only one thread bind is needed. But dpdk backend uses local listen table, we need to bind port for each worker. So if the first worker failed to bind, it could be think the normal error handle, like port is used. when assert happen, it means the first worker successfully to bind but the second worker failed, it's not expected and I can't think of the reason. So assert it